src/services/accounts/AccountsStakingPayoutsService.spec.ts

```diff
@@ -0,0 +1,282 @@
+import { ApiPromise } from '@polkadot/api';
+import { ApiDecoration } from '@polkadot/api/types';
+import { DeriveEraExposure } from '@polkadot/api-derive/types';
+import { Option, u32 } from '@polkadot/types';
+import {
+	AccountId32,
+	BalanceOf,
+	EraIndex,
+	Hash,
+} from '@polkadot/types/interfaces';
+import {
+	PalletStakingEraRewardPoints,
+	PalletStakingStakingLedger,
+	PalletStakingValidatorPrefs,
+} from '@polkadot/types/lookup';
+import { Codec } from '@polkadot/types/types';
+import { BadRequest } from 'http-errors';
+
+import { sanitizeNumbers } from '../../sanitize/sanitizeNumbers';
+import { polkadotRegistryV9122 } from '../../test-helpers/registries';
+import { defaultMockApi } from '../test-helpers/mock';
+import {
+	mockDeriveNominatedExposure,
+	mockDeriveValidatorExposure,
+	mockEraRewardPoints,
+	mockLedger,
+} from '../test-helpers/mock/accounts';
+import stakingPayoutsResponse from '../test-helpers/responses/accounts/stakingPayout.json';
+import { AccountsStakingPayoutsService } from './AccountsStakingPayoutsService';
+
+/**
+ * Addresses and data below were taken from era 533 around block ~7,760,000,
+ * on runtime v9122 Polkadot.
+ *
+ * The real world data has been modified to scale to the unit tests to act as mock data.
```

Comment:
How do you mean "modified to scale to the unit tests"?

Reply:
The original data set had all the validators and nominators for polkadot. I abstracted only 2 validators and 2 nominators so that the data is small and manageable, but also easy to configure and change for tests.

Reply:
ah, got it. Maybe just say "reduced to fit the unit tests"?

Reply:
Awesome switched it to "reduced to fit the unit tests", sounds better.